src/ontology/mondo-edit.obo

```diff
@@ -544720,7 +544720,7 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0100580
-name: epilepsy, onset unknown
```

Comment:
@Daniel-Olson can you please add this as a synonym (and you can add my ORCID as the source: https://orcid.org/0000-0001-5208-3432